.builder/actions/crt-ci-prep-xcodebuild.py

```diff
@@ -0,0 +1,9 @@
+import Builder
+
+class CrtCiPrepXCodebuild(Builder.Action):
```

Comment:
I haven't looked at the PR but can we avoid adding this action in crt-swift? We don't need to test the Cognito credential provider functionality in bindings; we can just create/destroy it for simple binding tests. The MQTT-specific tests should be in the IoT SDK.

Reply:
This script change is used to allow passing environment variables into xcodebuild tests. Regardless of the Cognito credential provider changes, we still need it for the unit tests that requires environment variables.